### DISCUSSION TITLE
CMU 计算机科学专业 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#39] [Announcements]
# CMU 计算机科学专业 | HackWay技术学习路线

卡内基梅隆大学CMU号称自己是世界上第一个开设计算机专业的学校（开设于1965年，另有说是普渡大学1962年），CMU的计算机堪称可以载入史册的化石级。

http://localhost:3000/docs/cs/path/berkeley

**Comment by xdlsy** [comment]
感谢

**Comment by miaoxu404** [comment]
感谢
